dev-python/webpy/webpy-0.40.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="a web framework for python that is as simple as it is powerful"
+HOMEPAGE="https://www.webpy.org"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-python/cheroot-6.0.0[${PYTHON_USEDEP}]
+"
+DEPEND="
+	${RDEPEND}
+	test? (
+		dev-python/dbutils[${PYTHON_USEDEP}]
+		>=dev-python/mysql-connector-python-8.0.19[${PYTHON_USEDEP}]
+		>=dev-python/psycopg-2.8.4[${PYTHON_USEDEP}]
+		>=dev-python/pymysql-0.9.3[${PYTHON_USEDEP}]
```

Comment:
This version restriction can be lifted off as it's always satisfied in ::gentoo.